core-api/src/controllers/access.controller.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from "express";
+import { AccessService } from "../services/access.service";
+import { HttpStatus } from "../utils/httpStatus";
+import { ServerError, ErrorCode } from "../utils/serverError";
+import Messages from "../utils/messages";
+
+export class AccessController {
+    public static async createAccess(req: Request, res: Response) {
+        const { place, permission } = req.body;
```

Comment:
The data coming from the request should be `place_id` and `permission_id`. Aside of checking if they are null, you should also check if they are `NaN`. Propagating raw strings to the service will make the database throw an error.

Reply:
Changes done